src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -204,6 +204,28 @@ class PythonSetup(Subsystem):
         ),
         advanced=True,
     )
+    no_binary = StrListOption(
+        "--no-binary",
+        help=(
+            "Do not use binary packages (i.e., wheels) for these 3rdparty distributions. Also "
+            "accepts :all: to disable all binary packages. Note that some packages are tricky to "
+            "compile and may fail to install when this option is used on them."
+            "See https://pip.pypa.io/en/stable/cli/pip_install/#install-no-binary for details."
+            'Note: Only takes effect if you use lockfiles, and set lockfile_generator = "pex" '
+            "in the [python] section of your config file."
+        ),
+    )
+    only_binary = StrListOption(
+        "--only-binary",
+        help=(
+            "Do not use source packages (i.e., sdists) for these 3rdparty distributions. Also "
+            "accepts :all: to disable all binary packages. Packages without binary distributions "
```

Comment:
... :all: to disable all source packages.

Reply:
Ooops.